store/mongo/datastore_mongo.go

```diff
@@ -428,6 +429,10 @@ func (db *DataStoreMongo) MigrateTenant(ctx context.Context, database, version s
 			ds:  db,
 			ctx: ctx,
 		},
+		&migration_1_9_0{
```

Comment:
Wouldn't it be reasonable to exclude `migration_1_1_0` since you're basically undoing it here?

Reply:
hmm we could in this case, but on principle I don't touch old migrations;
it's so easy to screw sth up for some particular user of a particular version...